scripts/inputs.py

```diff
@@ -104,7 +104,6 @@
 
 ###
 lut = [
```

Comment:
Why remove these column names? We won't remember what they are when we look again in 6 months. I very much recommend we keep them please.